src/ini-load.js

```diff
@@ -56,13 +86,78 @@ export function whenContentIniLoad(
     const promises = [];
     resources.forEach((r) => {
       if (!EXCLUDE_INI_LOAD.includes(r.element.tagName)) {
-        promises.push(r.loadedOnce());
+        promises.push(r.element.whenLoaded());
       }
     });
     return Promise.all(promises);
   });
 }
 
+/**
```

Comment:
Is the big picture reasons for async measure here that we don't need this to be "as soon as possible" timing? Some delays with the promise are okay

Reply:
There are several key elements:

1. The main time spent here would normally be waiting for an element to actually load. Even if this takes a little extra time to measure - it would be statistically drowned in the load-waiting time.
2. This is for analytics, so the less it affects the normal functioning of the system - the better. The previous version does the layout blocking calls. That creates a "measurement contaminates the result" situation.
3. For performance it's, of course, better to be able to have fewer blocking measurements.